kuma/users/newsletter/tasks.py

```diff
@@ -0,0 +1,37 @@
+from urllib.parse import quote
+
+from celery import task
+
+from kuma.users.models import User, UserSubscription
+
+from . import sendinblue
+
+
+@task
+def create_or_update_contact(user_pk):
+    user = User.objects.get(pk=user_pk)
+
+    if not user.is_newsletter_subscribed:
+        return
+
+    response = sendinblue.request(
+        "POST",
+        "contacts",
+        json={
+            "updateEnabled": True,
+            "email": user.email,
+            "attributes": {
+                "IS_PAYING": UserSubscription.objects.filter(
+                    user=user, canceled__isnull=True
+                ).exists()
+            },
+            "listIds": [int(sendinblue.LIST_ID)],
```

Comment:
```suggestion
            "listIds": [int(settings.SENDINBLUE_LIST_ID)],
```

Reply:
I'm still torn but I like consistency and your way is more consistent with what we're doing in other places. So I'll go with it! (not merging in the suggestions, because it needs some importing and moving yada yada)